public/app/features/explore/Logs.tsx

```diff
@@ -40,6 +40,8 @@ function renderMetaItem(value: any, kind: LogsMetaKind) {
         <LogLabels labels={value} />
       </span>
     );
+  } else if (kind === LogsMetaKind.Error) {
+    return <span className="logs-meta-item__error">{value}</span>;
```

Comment:
I wonder if a full width error message would not be better. But we can improve this in a later PR.

Reply:
👍